joplin/css/create_content_modal.scss

```diff
@@ -0,0 +1,251 @@
+  text-align: center;
+}
+
+.modal-dialog {
+  width: auto;
+}
+
+@media screen and (min-width: 50em) {
+  .modal-dialog {
+    padding: 0 0 2em;
+  }
+}
+
+// Content Steps
+
+.CreateContentModal__step {
+    padding: 0 20px 20px;
+}
+
+.CreateContentModal__step--active {
+  display: block !important;
+  padding: 20px;
+}
+
+.CreateContentModal__step--inactive {
+  display: none;
+}
+
+.CreateContentModal__header {
+  color: #023654;
+  font-size: 22px;
+  text-align: center;
+  line-height: 30px;
+}
+
+/* department header markup is hidden */
+.CreateContentModal__header--hidden {
+  display: none;
+}
+
+
+// CHOOSE TYPE STEP Component
+
+.ChooseTypeStep__options-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.ChooseTypeStep__option {
+  max-height: 200px;
+  border: 1px solid #EDEDED;
+  margin: 1px;
+  cursor: pointer;
+  flex: 1 0 48%;
+  max-width: calc(100% * (1/2) - 44px);
+  text-align: center;
+  color: #4A4A4A;
+  padding: 40px 20px 20px;
+  background: #FFFFFF;
+}
+
+.ChooseTypeStep__option:hover {
+  background: #EDEDED;
+}
+
+.ChooseTypeStep__option-header {
+  font-size: 14px;
+  font-weight: bold;
+}
+
+
+// CHOOSE TITLE STEP Component
+
+.ChooseTitleStep__input-label {
+  display: flex;
+  width: 100%;
+  font-weight: normal;
+}
+
+.ChooseTitleStep__input-label--left {
+  flex: 1;
+  width: 50%;
+  text-align: left;
+}
+
+.ChooseTitleStep__input-label--right {
+  flex: 1;
+  width: 50%;
+  text-align: right;
+}
+
+.ChooseTitleStep__input-label--red {
+  color: red;
+}
+
+.ChooseTitleStep__input-help {
+  font-size: 14px;
+  font-style: italic;
+  text-align: center;
+  padding: 10px 0 30px;
+  display: block;
+}
+
+.ChooseTitleStep__bullet-list {
+  padding-left: 20px;
+}
+
+.ChooseTitleStep__bullet-list li {
+  font-size: 14px;
+  text-align: left;
+  list-style: disc;
+}
+
+
+// CHOOSE TOPIC STEP Component
+
+.ChooseTopicStep__theme-group--closed {
+  display: none;
+}
+
+.ChooseTopicStep__theme-group--open {
+  display: block;
+}
+
+.ChooseTopicStep__theme-group {
+  padding: 10px 15%;
+  color: #3282AF;
+}
+
+.ChooseTopicStep__theme-toggle {
+  display: flex;
+}
+
+.ChooseTopicStep__minus {
+  background-image: url('../static/images/minus.png');
+  height: 21px;
+  width: 20px;
+  margin-right: 13px;
+}
+
+.ChooseTopicStep__plus {
+  background-image: url('../static/images/plus.png');
+  height: 21px;
+  width: 20px;
+  margin-right: 13px;
+}
+
+.ChooseTopicStep__topic-text {
+  font-size: 14px;
+}
+
+.ChooseTopicStep__topic-toggle {
+  display: flex;
+  margin: 10px 0;
+}
+
+.ChooseTopicStep__topic--unselected {
+  background-image: url('../static/images/unselected.png');
+  height: 22px;
+  width: 20px;
+  margin-right: 13px;
+  &:hover {
+    background-image: url('../static/images/grey_check.png');
+  }
+}
+
+.ChooseTopicStep__topic--selected {
+  background-image: url('../static/images/selected.png');
+  height: 22px;
+  width: 20px;
+  margin-left: 1px;
+  margin-right: 13px;
+}
+
+
+// BUTTON BAR COMPONENT
```

Comment:
might be nice to move these to their own files so you can just include the buttonbar styles in the buttonbar component etc

Reply:
✅